spec/requests/api/v1/spots/create_spec.rb

```diff
@@ -92,5 +92,15 @@
       end
       it_behaves_like 'invalid params'
     end
+
+    context 'when exceeding 10th spot creation limit' do
+      let!(:spots)          { create_list(:spot, 10, user_id: user.id) }
+      let(:params) do
+        {
+          spot: attributes_for(:spot, topic_id: topic.id, user_id: user.id)
+        }
+      end
+      it_behaves_like 'invalid params'
```

Comment:
It would be great if you can test that you are actually receiving the error message on the response too